scenario_runner.py

```diff
@@ -32,6 +32,8 @@
 
 import carla
 
+sys.path.append("C:/CARLA/CARLA_0.9.11/WindowsNoEditor/PythonAPI/carla") # Add to path in order to access BasicAgent, Localplanner etc.
```

Comment:
This path will only work for windows. This is a problem we have with all carla related stuff in virtual validation. @eriadam36  do you have any suggestion how we could adress this repository wise? Forking the entire carla rep?